server/src/main/scala/net/wiringbits/repositories/models/NotificationType.scala

```diff
@@ -0,0 +1,16 @@
+package net.wiringbits.repositories.models
+
+import enumeratum.EnumEntry.Uppercase
+import enumeratum.{Enum, EnumEntry}
+
+sealed trait NotificationType extends EnumEntry with Uppercase
+
+object NotificationType extends Enum[NotificationType] {
+  final case object EmailRegistration extends NotificationType
+  final case object VerifyAccount extends NotificationType
```

Comment:
`VerifyAccount`/`ResetPassword` won't get into the table, we need to take only the notifications that do not involve confidential data